frontend/src/views/LotteryView.vue

```diff
@@ -1,3 +1,61 @@
+<script setup lang="ts">
+import { ref, onMounted } from 'vue'
+import { useRoute } from 'vue-router'
+import { apiClient } from '@/api/apiClient'
+import type { components } from '@/api/schema'
+const ifDUplicated = ref(false)
+const route = useRoute()
+const eventId = route.params.eventId as string
+const lotteryId = route.params.lotteryId as string
+const lottery = ref<components['schemas']['Lottery'] | undefined>(undefined)
+const postLottery = async () => {
+  await apiClient.POST('/events/{eventID}/lotteries/{lotteryID}', {
+    params: {
+      path: { eventID: eventId, lotteryID: lotteryId },
+      query: { ifDuplicated: ifDUplicated.value }
+    },
+  })
+  await fetchLottery()
+}
+
+const fetchLottery = async () => {
+  const response = await apiClient.GET('/events/{eventID}/lotteries/{lotteryID}', {
+    params: { path: { eventID: eventId, lotteryID: lotteryId } }
+  })
+  lottery.value = response.data
+}
+
+onMounted(fetchLottery)
+</script>
 <template>
-  <v-container> Lottery </v-container>
+  <v-container max-width="600">
+    <v-card class="d-flex flex-column align-center ga-5">
+      <h1>
+        {{ lottery?.title }}
+      </h1>
+      <v-btn @click="postLottery"> 抽選スタート！！ </v-btn>
+      <v-checkbox
+      v-model="ifDUplicated"
+      :label="`イベント内の重複当選を許可する`"
+    ></v-checkbox>
+      <div class="d-flex">
+       
+      </div>
+      <div v-if="lottery !== undefined" class="d-flex ga-2">
+        <div>当選者 :</div>
```

Comment:
囲っている要素は文字列だけなので、`div`にする必要はなさそうです
後でスタイル当てることもあると思うので、`span`で囲うのが良いと思います
```suggestion
        <span>当選者 :</span>
```